docs/backups.md

```diff
@@ -0,0 +1,23 @@
+# OSBS Backups
+
+## Creating Backups
+
+Creating OSBS backups is very simple. To backup all relevant data, one needs to:
+
+1. Pause OSBS by `osbs pause-builds`
+2. Wait for all running builds to finish
+3. Backup data using following commands.
+  * `oc export -o=json BuildConfig > buildconfigs.json`
```

Comment:
Nice! Although I tend to think `-o json` or `--output=json` looks better than `-o=json`. Is it just me?